File: projects/m1/005-units-of-time-again/solutions/Andrea Andolfato/Units-of-time.py


time = float(input("Enter time in seconds: "))
day = time // (24*3600)
time = time % (24*3600)
hour = time // 3600
time %= 3600

min = time // 60
time %= 60
seconds = time
print("d:h:m:s-> %d:%d:%d:%d" %  (day , hour , min, seconds ,))

File: projects/m1/004-units-of-time/solutions/alessiacaporaso/units-of-time.js
let secondsInDays = 86400;
let secondsInHours = 3600;
let secondsInMinutes = 60;
// enter day, hours, minutes and second
let userChoiceDay = parseInt(prompt("Please, enter the number of Days: "));
let userChoiceHours = parseInt(prompt("Please, enter the number of Hours: "));
let userChoiceMinutes = parseInt(prompt("Please, enter the number of Minutes: "));
let userChoiceSeconds = parseInt(prompt("Please, enter the number of Seconds: "));
//calc tot seconds
let totSeconds = (userChoiceDay * secondsInDays) + (userChoiceHours * secondsInHours) + (userChoiceMinutes * secondsInMinutes) + userChoiceSeconds;

alert(`Namber of seconds is: ${totSeconds}`);